### DISCUSSION TITLE
No data

### DISCUSSION THREAD

**Discussion by Julien4135** [tillsteinbach/VWsFriend#511] [Q&A]
Hello, since 10/06 I have nothing coming back to the car.
Everything is up to date on my side and I haven't seen any new conditions to validate (Cookies).

![image](https://github.com/tillsteinbach/VWsFriend/assets/93113346/2531c900-7412-4974-b2e2-9a9e731532ac)

**Comment by tillsteinbach** [comment]
What does the log output say? Does a restart solve the problem?

**Reply by Julien4135** [reply]
Rebooting does nothing. 
The newspaper never stops scrolling

![image](https://github.com/tillsteinbach/VWsFriend/assets/93113346/b8d40af9-fea1-4044-ac80-979064dbc792)
![image](https://github.com/tillsteinbach/VWsFriend/assets/93113346/142fdddc-b0ec-47d4-b48c-bb69007ebac4)

**Comment by Julien4135** [comment]
Is the problem with me? 

is everything working again now?
